openest/generate/calculation.py

```diff
@@ -29,6 +29,7 @@ def format(self, lang, *args, **kwargs):
         """Returns a dictionary of FormatElements.
         Only keys in the tree of dependencies will be output.
         """
+        print(self.__class__)
```

Comment:
A wanted to leave this in as a permanent debugging aid. Any time we fall back on the `Calculation` call to `format`, it means that we never implemented the function (so we throw there a `NotImplementedError`). But we often don't know which class is triggering this, because we have been in the habit of adding new classes without providing a `format` implementation, since we don't always use that subsystem.

I've cleaned it up a little.